Modules/Office365DSC/Modules/O365DSCLogEngine.psm1

```diff
@@ -22,8 +22,8 @@ function New-Office365DSCLogEntry
 
     try
     {
-        $VerbosePreference = 'Continue'
```

Comment:
how come this was deleted?

Reply:
Because this would override what we set as preference before